packages/orbit-components/src/Stepper/StepperStateless/index.tsx

```diff
@@ -115,8 +119,8 @@ const StepperStateless = ({
             onIncrement(ev);
           }
         }}
-        title={titleIncrement}
         icons={iconStyles}
+        title={titleIncrement}
```

Comment:
I was thinking of changing this prop name from `title` to `ariaLabel` and making it more specific, however, this would be (probably) BC. I searched where this prop is used and I've found it's used [here](https://gitlab.skypicker.com/frontend/home/-/blob/main/src/search/scenes/HomePage/components/UnfinishedBookingBanner/NewDesktopVersion/UnfinishedBookingButtonDesktop.tsx).

I'm not sure if it makes sense to change this prop name, if it could be included within this PR or in a separate one.

Reply:
No, that wouldn't make sense. :D  My suggestion was to rename _aria_ attribute in `ButtonPrimitive` (probably in a different PR) and also change `titleIncrement` (`titleDecrement`). I kept `titleDecrement` (`titleIncrement`) because of attribute naming in ButtonPrimitive.

As you said, I also think it's out of the scope of this PR, that's the reason why I didn't touch ButtonPrimitive at all.